anago

```diff
@@ -742,6 +743,23 @@ get_build_candidate () {
                                 $PARENT_BRANCH \
    || return 1
 
+  # When creating alphas, make sure that X.X.0 has been created first or warn.
```

Comment:
just thinking about this more, we should allow X.Y+1.0.alpha.N as soon as code slush has ended and we're no longer going to fast-forward the release branch.

I'm not sure if there's a way to encode this in the tool.